src/pool/srv_pool.c

```diff
@@ -292,6 +292,41 @@ pool_svc_rdb_uuid_remove(const uuid_t pool_uuid)
 	return rc;
 }
 
+void
+ds_pool_destroy_pl_map(struct ds_pool *pool)
+{
+	if (pool->sp_pl_map != NULL) {
+		struct pl_map	*map = pool->sp_pl_map;
+		bool		 last;
+
+		pool->sp_pl_map = NULL;
+		D_SPIN_LOCK(&map->pl_lock);
+		/* Drop the reference when create */
+		map->pl_ref--;
+		last = (map->pl_ref == 0);
+		D_SPIN_UNLOCK(&map->pl_lock);
```

Comment:
This pl_ref--  might need moved to placement module, either pl_map_destroy or create pl_map_decref()?

Reply:
OK, I will adjust the code.